files/scripts/supervisor-proc-exit-listener

```diff
@@ -10,14 +10,28 @@ import swsssdk
 
 from supervisor import childutils
 
-# Contents of file should be the names of critical processes (as defined in
-# supervisor.conf file), one per line
+# Contents of file should be the names of critical processes and group name 
+# (as defined in supervisor.conf file), one per line
 CRITICAL_PROCESSES_FILE = '/etc/supervisor/critical_processes'
 
 # This table in databse contains the features for container and each
 # feature for a row will be configured a state or number.
 CONTAINER_FEATURE_TABLE_NAME = 'CONTAINER_FEATURE'
 
+# Read the list of critical processes and group names from a file
+def get_group_list_and_critical_processes():
+    group_list = []
+    critical_processes = [] 
```

Comment:
Prefer consistent naming. Either `critical_group_list` and `critical_process_list` *or* `critical_groups` and `critical_processes`

Reply:
Updated.